flow/matrix_decomposition.cc

```diff
@@ -28,6 +28,13 @@ static inline SkVector3 SkVector3Cross(const SkVector3& a, const SkVector3& b) {
 MatrixDecomposition::MatrixDecomposition(const SkMatrix& matrix)
     : MatrixDecomposition(SkMatrix44{matrix}) {}
 
+static inline void normalize(SkVector3& v) {
```

Comment:
Consider making the method name more descriptive. Also, refer to the style guide for capitalization (go/fluttercpp). Something like `SkVector3Normalize`.